src/components/LoadingIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import { type NextComponentType } from "next";
+
+const LoadingIcon: NextComponentType = () => {
+  return (
+    <div className="flex h-screen justify-center">
+      <div className="relative h-24 w-24 animate-spin rounded-full bg-gradient-to-r from-blue-500 to-purple-500">
```

Comment:
I likes!

Reply:
Thanks! It was a lot of fun and a good refresher on setting up a new dev environment again.  Worked instantly when I downloaded and used docker desktop rather than trying with Homebrew